backend/app/views/spree/admin/orders/confirm/_shipment.html.erb

```diff
@@ -11,11 +11,11 @@
   <table class="stock-contents index" data-hook="stock-contents">
     <thead>
       <th colspan="2">
-        <%= Spree.t(:item_description) %>
+        <%= Spree::LineItem.human_attribute_name(:description) %>
```

Comment:
Yes, I used `LineItem` as new home for `item_description`. Looking at the table, you will see there are items described